src/test/java/com/github/dockerjava/test/serdes/JSONTestHelper.java

```diff
@@ -25,14 +25,14 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.github.dockerjava.api.command.CommandJSONSamples;
 
 /**
  * Provides helper methods for serialization-deserialization tests
  * 
  * @author Oleg Nenashev
- * @since TODO
  */
+//TODO: Create helper that loads json files from simple folder structure using a type,
```

Comment:
better place in javadoc, more chances that somebody will see it :) Though i plan change it soon.